### PR TITLE
test: Fedora 26 has branched start testing there

### PR DESCRIPTION
**Pull request by stefwalter**
Now that Fedora 26 has branched, we won't be releasing into the main Fedora 25 channels any longer.
    
Remove support for testing on Fedora 24 as well, and move Fedora 25 testing to a more advisory role, where it's not in every pull request.
    
Certain things have not been migrated away from Fedora 24 where they use it as a base for a different kind of image. That can happen later.

### CODE REVIEW

File: test/images/scripts/fedora-26.bootstrap [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301 USA.
+
+BASE=$(dirname $0)
+$BASE/virt-install-fedora "$1" x86_64 "http://dl.fedoraproject.org/pub/fedora/linux/releases/25/Server/x86_64/os"
```
Comment (dperpeet) [inline test/images/scripts/fedora-26.bootstrap]: Shouldn't we wait with this until we have actual fedora 26 packages? I don't see any enablement in the setup file either that would actually make this "fedora 26".

Comment (mvollmer) [inline test/images/scripts/fedora-26.bootstrap]: This should probably be `http://dl.fedoraproject.org/pub/fedora/linux/development/26/Server/x86_64/os/`

File: test/images/fedora-25 [hunk 2]
```diff
@@ -1 +1 @@
-fedora-25-e9d6eb23276c829e39297939534ace20696d0641.qcow2
\ No newline at end of file
+fedora-25-1ca049fdc5c133be62b3ace0e91834bcba577fde.qcow2
```
Comment (mvollmer) [inline test/images/fedora-25]: Why has the fedora-25 image changed?

File: test/images/scripts/fedora-26.setup [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+# Setup basics for building images
+docker build -t cockpit/base /var/tmp/cockpit-base
+
+# Configure kubernetes
+/var/lib/testvm/kubernetes.setup
+
+# docker images that we need for integration testing
+/var/lib/testvm/docker-images.setup
+
+# HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1341829
```
Comment (mvollmer) [inline test/images/scripts/fedora-26.setup]: This is supposedly fixed in Fedora 26, so we can try removing this hack.

**Comment by mvollmer** [comment]
Continued in #6069

**Pull request closed without merging**